packages/paste-website/src/pages/foundations/content/voice-and-tone/index.mdx

```diff
@@ -38,187 +38,65 @@ export const getStaticProps = async () => {
 
 <content>
 
-## What are voice and tone?
+How to use the Twilio voice and tonal qualities to communicate in our products.
```

Comment:
```suggestion
```

Reply:
This can be removed since it's repeated in the page description: https://paste-docs-git-megan-content-updates-1-twilio.vercel.app/foundations/content/voice-and-tone